src/Equinox.MessageDb/MessageDb.fs

```diff
@@ -210,23 +233,28 @@ module Read =
             "Mdb{action:l} stream={stream} count={count}/{batches} version={version}",
             action, streamName, count, batches, version)
 
-    let private logLastEventRead streamName t events version (log: ILogger) =
+    let private logLastEventRead (span: Activity) streamName t events (version: int64) (log: ILogger) =
         let bytes = resolvedEventBytes events
         let count = events.Length
         let reqMetric : Log.Measurement = { stream = streamName; interval = t; bytes = bytes; count = count}
         let evt = Log.Metric.ReadLast reqMetric
+        if span <> null then span.AddTag("eqx.bytes", bytes).AddTag("eqx.count", count).AddTag("eqx.version", version) |> ignore
         (log |> Log.prop "bytes" bytes |> Log.event evt).Information(
             "Mdb{action:l} stream={stream} count={count} version={version}",
             "ReadL", streamName, count, version)
 
     let internal loadLastEvent (log : ILogger) retryPolicy (reader : MessageDbReader) requiresLeader streamName
         : Async<int64 * ITimelineEvent<EventBody> array> = async {
+        use span = Tracing.source.StartActivity("ReadLast")
+        if span <> null then
+            span.AddTag("eqx.stream_name", streamName) |> ignore
+            if requiresLeader then span.AddTag("eqx.requires_leader", true) |> ignore
```

Comment:
extract as `Tracing.requiresLeader requiresLeader` or similar (see suggestions on main tracing PR)

Reply:
I've extracted these into the core equinox internal module as extension methods

Reply:
🤦 Can't move it into Equinox.Core because that screws the build up because Activity was added after netstandard2.1

Reply:
I've made a Tracing.fs file instead. If/when we add tracing to more stores let's move it from there into some common place